From 73fdb5acf298a89027075fe14493077d172db834 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Fri, 7 Feb 2020 22:10:49 +0100
Subject: [PATCH 1/1] efi_loader: fix efi_install_fdt() description

In the function description use the correct parameter name.

Mention EFI_FDT_USE_INTERNAL.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 cmd/bootefi.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/cmd/bootefi.c b/cmd/bootefi.c
index 56bdff33c6..a030d14dc8 100644
--- a/cmd/bootefi.c
+++ b/cmd/bootefi.c
@@ -202,15 +202,15 @@ static void *get_config_table(const efi_guid_t *guid)
 /**
  * efi_install_fdt() - install device tree
  *
- * If fdt_addr is available, the device tree located at that memory address will
- * will be installed as configuration table, otherwise the device tree located
- * at the address indicated by environment variable fdt_addr or as fallback
- * fdtcontroladdr will be used.
+ * If fdt is not EFI_FDT_USE_INTERNAL, the device tree located at that memory
+ * address will will be installed as configuration table, otherwise the device
+ * tree located at the address indicated by environment variable fdt_addr or as
+ * fallback fdtcontroladdr will be used.
  *
  * On architectures using ACPI tables device trees shall not be installed as
  * configuration table.
  *
- * @fdt_addr:	address of device tree or EFI_FDT_USE_INTERNAL to use the
+ * @fdt:	address of device tree or EFI_FDT_USE_INTERNAL to use the
  *		the hardware device tree as indicated by environment variable
  *		fdt_addr or as fallback the internal device tree as indicated by
  *		the environment variable fdtcontroladdr
-- 
2.24.1

